proposals/compatibility-mode.md

```diff
@@ -192,26 +192,34 @@ compatibility mode.
 
 ## 18. Introduce new `maxStorageBuffersInVertexStage` and `maxStorageTexturesInVertexStage` limits.
 
-If the number of shader variables of type `storage_buffer` in a vertex shader exceeds the `maxStorageBuffersInVertexStage` limit, a validation error will occur at pipeline creation time.
+In `createBindGroupLayout`, `createPipelineLayout` and at pipeline creation time, if the number of bindings of type `storage_buffer` with visibility `VERTEX` bit true, exceeds the `maxStorageBuffersInVertexStage` limit a validation error will occur.
```

Comment:
nit:

> at pipeline creation time

I think this is implied, because:
- if using explicit layouts, it already failed at `createPipelineLayout`
- if using `"auto"` layouts, it will fail when it calls `createPipelineLayout` internally

Just to clarify that, could say something like:

```suggestion
In `createBindGroupLayout` and `createPipelineLayout` (including `"auto"` layout creation), if the number of bindings of type `storage_buffer` with visibility `VERTEX` bit true, exceeds the `maxStorageBuffersInVertexStage` limit a validation error will occur.
```

Same elsewhere.